.gitignore

```diff
@@ -50,8 +50,12 @@ generated
 *.zip
 
 *.js
+!.eslintrc.js
```

Comment:
Why do we want to ignore this file?

Reply:
The ! makes this file to never be ignored. It's just an enforcement.